docs/admin/kubernetes_deployment.md

```diff
@@ -1,52 +1,55 @@
 # Kubernetes Deployment Installation Guide
 KServe supports `RawDeployment` mode to enable `InferenceService` deployment with Kubernetes resources [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment), [`Service`](https://kubernetes.io/docs/concepts/services-networking/service), [`Ingress`](https://kubernetes.io/docs/concepts/services-networking/ingress) and [`Horizontal Pod Autoscaler`](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale). Comparing to serverless deployment it unlocks Knative limitations such as mounting multiple volumes, on the other hand `Scale down and from Zero` is not supported in `RawDeployment` mode.
 
-Kubernetes 1.22 is the minimally required version and please check the following recommended Istio versions for the corresponding
```

Comment:
not sure if we need to remove it.

Reply:
My understanding was that Aston was not required ```The current installation guide for raw deployment mode is not accurate. For example, Istio is not needed if disableIngressCreation is enabled. See https://github.com/kserve/kserve/pull/3436```

Reply:
Yes, raw does not require it, but for the serverless mode the docs advises to use it, iirc.